lifelines/fitters/aalen_johansen_fitter.py

```diff
@@ -184,17 +179,16 @@ def _bounds(self, lagged_survival, alpha, ci_labels):
         for _, r in df.iterrows():
             sf = df.loc[df.index <= r.name].copy()
             F_t = float(r["Ft"])
-            sf["part1"] = ((F_t - sf["Ft"]) ** 2) * (
-                sf["observed"] / (sf["at_risk"] * (sf["at_risk"] - sf["observed"]))
-            )
-            sf["part2"] = (
-                ((sf["lagS"]) ** 2)
-                * sf[self.label_cmprisk]
-                * ((sf["at_risk"] - sf[self.label_cmprisk]))
-                / (sf["at_risk"] ** 3)
-            )
-            sf["part3"] = (F_t - sf["Ft"]) * sf["lagS"] * (sf[self.label_cmprisk] / (sf["at_risk"] ** 2))
-            variance = (np.sum(sf["part1"])) + (np.sum(sf["part2"])) - 2 * (np.sum(sf["part3"]))
+            variance = np.sum((F_t - sf["Ft"]) ** 2 *
```

Comment:
did `black` do this???

Reply:
If you are referring to `variance` being collapsed into a single calculation (rather than in three pieces) that was my change.

The spacing may either be `black` or my IDE (PyCharm)

Reply:
it's a really difficult to read piece of code. Can you break it back up a bit by using intermediate values? For example: https://github.com/CamDavidsonPilon/lifetimes/blob/master/lifetimes/fitters/beta_geo_fitter.py#L156-L166